pokedex/data/csv/pokemon.csv

```diff
@@ -905,61 +996,96 @@ id,identifier,species_id,height,weight,base_experience,order,is_default
 10097,pikachu-unova-cap,25,4,60,112,45,0
 10098,pikachu-kalos-cap,25,4,60,112,46,0
 10099,pikachu-alola-cap,25,4,60,112,47,0
-10100,raichu-alola,26,7,210,218,50,0
-10101,sandshrew-alola,27,7,400,60,52,0
-10102,sandslash-alola,28,12,550,158,54,0
-10103,vulpix-alola,37,6,99,60,65,0
-10104,ninetales-alola,38,11,199,177,67,0
-10105,diglett-alola,50,2,10,53,83,0
-10106,dugtrio-alola,51,7,666,149,85,0
-10107,meowth-alola,52,4,42,58,87,0
-10108,persian-alola,53,11,330,154,89,0
-10109,geodude-alola,74,4,203,60,113,0
-10110,graveler-alola,75,10,1100,137,115,0
-10111,golem-alola,76,17,3160,223,117,0
-10112,grimer-alola,88,7,420,65,133,0
-10113,muk-alola,89,10,520,175,135,0
-10114,exeggutor-alola,103,109,4156,186,153,0
-10115,marowak-alola,105,10,340,149,156,0
-10116,greninja-battle-bond,658,15,400,239,766,0
-10117,greninja-ash,658,15,400,288,767,0
-10118,zygarde-10,718,12,335,219,836,0
-10119,zygarde-50,718,50,3050,270,837,0
-10120,zygarde-complete,718,45,6100,319,838,0
-10121,gumshoos-totem,735,14,600,146,858,0
-10122,vikavolt-totem,738,26,1475,225,862,0
-10123,oricorio-pom-pom,741,6,34,167,866,0
-10124,oricorio-pau,741,6,34,167,867,0
-10125,oricorio-sensu,741,6,34,167,868,0
-10126,lycanroc-midnight,745,11,250,170,875,0
-10127,wishiwashi-school,746,82,786,217,878,0
-10128,lurantis-totem,754,15,580,168,888,0
-10129,salazzle-totem,758,21,810,168,893,0
-10130,minior-orange-meteor,774,3,400,154,910,0
-10131,minior-yellow-meteor,774,3,400,154,911,0
-10132,minior-green-meteor,774,3,400,154,912,0
-10133,minior-blue-meteor,774,3,400,154,913,0
-10134,minior-indigo-meteor,774,3,400,154,914,0
-10135,minior-violet-meteor,774,3,400,154,915,0
-10136,minior-red,774,3,3,175,916,0
-10137,minior-orange,774,3,3,175,917,0
-10138,minior-yellow,774,3,3,175,918,0
-10139,minior-green,774,3,3,175,919,0
-10140,minior-blue,774,3,3,175,920,0
-10141,minior-indigo,774,3,3,175,921,0
-10142,minior-violet,774,3,3,175,922,0
-10143,mimikyu-busted,778,2,7,167,928,0
-10144,mimikyu-totem-disguised,778,4,28,167,929,0
-10145,mimikyu-totem-busted,778,4,28,167,930,0
-10146,kommo-o-totem,784,24,2075,270,937,0
-10147,magearna-original,801,10,805,270,958,0
+10100,raichu-alola,26,7,210,243,51,0
+10101,sandshrew-alola,27,7,400,60,53,0
+10102,sandslash-alola,28,12,550,158,55,0
+10103,vulpix-alola,37,6,99,60,66,0
+10104,ninetales-alola,38,11,199,177,68,0
+10105,diglett-alola,50,2,10,53,84,0
+10106,dugtrio-alola,51,7,666,149,86,0
+10107,meowth-alola,52,4,42,58,88,0
+10108,persian-alola,53,11,330,154,91,0
+10109,geodude-alola,74,4,203,60,115,0
+10110,graveler-alola,75,10,1100,137,117,0
+10111,golem-alola,76,17,3160,223,119,0
+10112,grimer-alola,88,7,420,65,141,0
+10113,muk-alola,89,10,520,175,143,0
+10114,exeggutor-alola,103,109,4156,186,161,0
+10115,marowak-alola,105,10,340,149,164,0
+10116,greninja-battle-bond,658,15,400,239,787,0
+10117,greninja-ash,658,15,400,288,788,0
+10118,zygarde-10-power-construct,718,12,335,243,858,0
+10119,zygarde-50-power-construct,718,50,3050,300,859,0
```

Comment:
i changed those two identifiers (and added an additional `zygarde-10` below) to better represent the in-game data (2 "standard" formes, 2 power construct formes, 1 zygarde complete).
Both here and in `pokemon_forms.csv`